README.md

```diff
@@ -32,9 +34,87 @@ For alternative ways, stop by our Community Hub [stackrox.io](https://www.stackr
     + [How to Deploy](#how-to-deploy)
 * [Generating portable installers](#generating-portable-installers)
 
-## Deployment
+## Quick Installation via Helm
 
-To quickly deploy the latest development version of StackRox to your kubernetes
+StackRox offers quick installation via Helm Charts. Follow the [Helm Installation Guide](https://helm.sh/docs/intro/install/) to get `helm` CLI on your system.
+First, add the [stackrox/helm-charts/opensource](https://github.com/stackrox/helm-charts/tree/main/opensource) repository to Helm.
+```sh
+helm repo add stackrox https://raw.githubusercontent.com/stackrox/helm-charts/main/opensource/
+```
+To see all available Helm charts in the repo run (you may add the option `--devel` to show non-release builds as well)
+```sh
+helm search repo stackrox
+```
+In order to install stackrox-central-services you will need a secure password. This password will be needed later when creating an init bundle.
+```sh
+openssl rand -base64 20 | tr -d '/=+' > stackrox-admin-password.txt
+```
+
+From here you can install stackrox-central-services to get Central and Scanner components deployed on your cluster. Note that you need only one deployed instance of stackrox-central-services even if you plan to secure multiple clusters.
+```sh
+helm install -n stackrox --create-namespace stackrox-central-services stackrox/stackrox-central-services --set central.adminPassword.value="$(cat stackrox-admin-password.txt)"
+```
+
+If you're deploying StackRox on a small node like a local development cluster, run the following command to reduce StackRox resource requirements. Please keep in mind that these reduced resource settings are not suited for production setups.
+```sh
+helm upgrade -n stackrox stackrox-central-services stackrox/stackrox-central-services \
+  --set central.resources.requests.memory=1Gi \
+  --set central.resources.requests.cpu=1 \
+  --set central.resources.limits.memory=4Gi \
+  --set central.resources.limits.cpu=1 \
+  --set scanner.autoscaling.disable=true \
+  --set scanner.replicas=1 \
+  --set scanner.resources.requests.memory=500Mi \
+  --set scanner.resources.requests.cpu=500m \
+  --set scanner.resources.limits.memory=2500Mi \
+  --set scanner.resources.limits.cpu=2000m
+```
+To create a secured cluster, you first need to generate an init bundle containing initialization secrets. The init bundle will be saved in `stackrox-init-bundle.yaml`. Use it to provision secured clusters as shown below.
+```sh
+kubectl -n stackrox exec deploy/central -- roxctl --insecure-skip-tls-verify \
+  --password "$(cat stackrox-admin-password.txt)" \
+  central init-bundles generate stackrox-init-bundle --output - > stackrox-init-bundle.yaml
+```
+Then install stackrox-secured-cluster-services in the same cluster using this command with the init bundle you just generated:
+```sh
+helm install -n stackrox stackrox-secured-cluster-services stackrox/stackrox-secured-cluster-services \
+  -f stackrox-init-bundle.yaml \ 
```

Comment:
Please do not leave whitespace after `\` character, this breaks the command.

Reply:
Good to know, I was wondering why it broke last time I copied it over but didn't figure out why.